lib/ravelin/event.rb

```diff
@@ -50,12 +50,12 @@ def validate_top_level_payload_params
         when :login
           validate_payload_inclusion_of :customer_id, :temp_customer_id
         when :checkout
-          validate_payload_inclusion_of :customer, :order, :items,
+          validate_payload_inclusion_of :order,
```

Comment:
We still want to validate the presence of a `:customer` object, even if it just includes the `:customer_id`